python/setup.py

```diff
@@ -205,7 +205,8 @@ def get_cuda_version_from_header(cuda_include_dir):
         "Programming Language :: Python :: 3.7",
     ],
     # Include the separately-compiled shared library
-    setup_requires=["cython"],
+    setup_requires=["Cython>=0.29,<0.30"],
+    extra_requires=["pytest", "pytest-xdist"],
```

Comment:
This should be `extras_require` and be a dictionary: https://hanxiao.io/2019/11/07/A-Better-Practice-for-Managing-extras-require-Dependencies-in-Python/

Reply:
We could also technically move this to the `setup.cfg` as well, but let's do that in a follow up.

Reply:
> This should be `extras_require` and be a dictionary: https://hanxiao.io/2019/11/07/A-Better-Practice-for-Managing-extras-require-Dependencies-in-Python/

Fixed this 👍